src/Controls/src/Core/ContentPresenter.cs

```diff
@@ -19,12 +21,38 @@ public class ContentPresenter : Compatibility.Layout, IContentView
 		/// <include file="../../docs/Microsoft.Maui.Controls/ContentPresenter.xml" path="//Member[@MemberName='.ctor']/Docs/*" />
 		public ContentPresenter()
 		{
-			this.SetBinding(
+			SetBinding(
```

Comment:
The test **RadioButtonWithValueChangeSelected** is failing on iOS.
![image](https://github.com/user-attachments/assets/7843b5e1-ced6-4d61-87f9-25e4d6cf607e)
Is not rendering the **Content**, the text.
![image](https://github.com/user-attachments/assets/2a7ea9fb-0871-45d8-8590-fef6ea8d6865)